src/main/java/org/openstreetmap/atlas/checks/event/FileProcessor.java

```diff
@@ -159,8 +160,8 @@ public void setBatchSize(final int batchSize)
      */
     protected String getFilename()
     {
-        return String.format("%s-%s%s", new Date().getTime(), this.getCount(),
-                new LogFilePathFilter(this.compressOutput).getExtension());
+        return String.format("%sP%s-%s%s", new Date().getTime(), TaskContext.getPartitionId(),
```

Comment:
Is there any chance this may cause issues with MR upload?

Reply:
All of the utilities I am aware of only care about the folder structure and the file extension of the flag outputs. The MR upload command should be unaffected by this change.